<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewsGroupDetail</class>
 <widget class="QWidget" name="NewsGroupDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewsGroupDetail</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="m_serverList">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>News Server</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Low Water Mark</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>High Water Mark</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Article Parts Available</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last Updated</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshServersButton">
       <property name="text">
        <string>&amp;Refresh</string>
       </property>
       <property name="icon">
        <iconset resource="quban.qrc">
         <normaloff>:/quban/images/fatCow/Update.png</normaloff>:/quban/images/fatCow/Update.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="m_localServer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Local Server</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Low Water Mark</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>High Water Mark</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Article Parts Held</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="getPartsGroupBox">
     <property name="title">
      <string>Get Article Parts</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="getFromButton">
               <property name="text">
                <string>From</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="getFromVal"/>
             </item>
             <item>
              <widget class="QLabel" name="toLabel">
               <property name="text">
                <string>to</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="getToVal"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="getAllButton">
             <property name="text">
              <string>All</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="getLastButton">
               <property name="text">
                <string>Last</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="getLastVal"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QRadioButton" name="getNextButton">
               <property name="text">
                <string>First / Next</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="getNextVal"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="expireGroupBox">
     <property name="title">
      <string>Article Retention</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QRadioButton" name="expireByServButton">
             <property name="text">
              <string>Expire articles no longer on server(s)</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QRadioButton" name="expireByDateButton">
               <property name="text">
                <string>Expire articles over</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="expireDaysVal"/>
             </item>
             <item>
              <widget class="QLabel" name="expireDaysLabel">
               <property name="text">
                <string>days</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QRadioButton" name="expireByIdButton">
               <property name="text">
                <string>Expire articles with id earlier than</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="expireIdVal"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="expireAllButton">
             <property name="text">
              <string>Expire  all</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="expiryDate">
            <property name="text">
             <string>Date expiry last run</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lastExpiryDate">
            <property name="maxLength">
             <number>11</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="quban.qrc"/>
 </resources>
 <connections/>
</ui>
